### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by taeram** [AtomLinter/linter-php#19]
As reported by the Deprecation Cop in Atom 0.196.0:

Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (C:\Atom\resources\app.asar\src\package.js:790:9)
Package.hasActivationCommands (C:\Atom\resources\app.asar\src\package.js:715:20)
<unknown> (C:\Atom\resources\app.asar\src\package.js:169:24)
Package.measure (C:\Atom\resources\app.asar\src\package.js:147:15)
```

**Comment by joshbadger** [comment]
I'm experiencing the exact same issue.

**Comment by kevinsawicki** [comment]
Fixed by #29  and published in `0.0.13`
